package.json

```diff
@@ -2,6 +2,10 @@
   "name": "@opencrvs/countryconfig",
   "version": "1.3.3",
   "description": "OpenCRVS country configuration for reference data",
+  "os": [
+    "darwin",
+    "linux"
+  ],
```

Comment:
Why this addition? Would `engines` definition be better so we can restrict the node.js version while at least not explicitly blocking Windows platforms?

Reply:
It was mentioned in the issue description and also as far as I know we do not support development in windows. Thus I thought it would be a step to make that point more explicit. I also thought about making it `"os": ["!win32"]`. What do you think @rikukissa ?

Reply:
Let's keep it as you have it now @tahmidrahman-dsi 👍 better to explicitly list out the ones we support

Reply:
I can then later create a PR in case I have issues with WSL. This is fine to me as well.